### DISCUSSION TITLE
No Connection

### DISCUSSION THREAD

**Discussion by Gringo84** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#53] [Q&A] [answered]
Hey Guys,

first i would thank you for the great guide about the Lay-Z-Spa Wifi remote.

Its my first arduino programming and I think i am not far away from the goal but I have the following problem:

After entering the wifi credentials over the Auto Portal, i cannot get any Connection over the IP adress which my Router has given to the ESP8266. My last action was to enter these adress into the global sketch and upload it to the ESP8266. But there is still no connection.

Maybe you have an idea, what my problem could be?

Greetings and a happy new year to everyone

**Comment by visualapproach** [comment, marked as answer]
Hi, did you upload the data files with LittleFS tool? If your router gave it an IP there is likely a connection. But with no files on the ESP you should get a 404 error.

**Comment by Gringo84** [comment]
This was it. I only uploaded the LittleFS Sketch and not the Files. Now i can open it.
Thank you. Now i have to doublecheck the board, there must be a wiring mistake. On the Lay-Z-Spa Pump nothing works....
